packs/data/feats.db/gravelands-herbalist.json

```diff
@@ -24,9 +24,6 @@
             "value": [
                 {
                     "value": "Herbalist Dedication"
-                },
-                {
-                    "value": "trained in Religion"
```

Comment:
Why was this removed? Trained in Religion *is* a requirement

Reply:
It does not show Trained in Religion as a requirement for Gravelands Herbalist on AON https://2e.aonprd.com/Feats.aspx?ID=3614, which is where I was looking. I'll add it back.

Reply:
Fixed in commit [e58fa08](https://github.com/foundryvtt/pf2e/pull/2596/commits/e58fa08ddcd4be0ff493652b4f8a9085a789fd65)